### ISSUE TITLE
Getting 403 when trying to access the /v2/admin/location/states api.

### ISSUE THREAD

**Issue by KINGpro-beep** [cowinapi/developer.cowin#347]
I am making a call out using the Salesforce Apex Rest service , but it is giving me an the code of 403 along with the error 
" We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner."
and providing me with a request id and when I click on the "Contact Us " hyperlink then simply gives error "
Fetch errorNot Found /api_specification_v10/contactus.php.yaml".

**Comment by Bhartendu-Kumar** [comment]
Is the call coming from Indian IP?

**Comment by KINGpro-beep** [comment]
No, it's a Salesforce Org which is running on the US servers I think , but
definitely it's not the Indian IP.
Thanks

**Comment by mshaheer92** [comment]
I'm getting the same error when I'm trying it from an android device. How do I fix this? Is this a geo location problem @Bhartendu-Kumar ?
